sense_hat_example/sense_stick.js

```diff
@@ -47,6 +53,22 @@ function onLoad(event) {
                 return;
         }
 
+        switch(state) {
+            case SenseStickDevice.STATE_PRESS:
+                if (Sk.sense_hat.sensestick.isKeyDown(e.key)) {
```

Comment:
@waywaaard Can we rely on `e.key`? Maybe I'm reading [this](http://caniuse.com/#feat=keyboardevent-key) wrong but it looks like Safari, among others, doesn't support it. But of course [this](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent) says it **should** be used and that `e.which` (this is what I'm currently using) is deprecated.